src/Umbraco.Web.UI.Client/src/views/common/dialogs/help.controller.js

```diff
@@ -1,14 +1,21 @@
 angular.module("umbraco")
-    .controller("Umbraco.Dialogs.HelpController", function ($scope, $location, $routeParams, helpService, userService) {
+    .controller("Umbraco.Dialogs.HelpController", function ($scope, $location, $routeParams, helpService, userService, localizationService) {
         $scope.section = $routeParams.section;
         $scope.version = Umbraco.Sys.ServerVariables.application.version + " assembly: " + Umbraco.Sys.ServerVariables.application.assemblyVersion;
         
         if(!$scope.section){
-            $scope.section ="content";
+            $scope.section = "content";
         }
 
+        $scope.sectionName = $scope.section;
+
         var rq = {};
         rq.section = $scope.section;
+
+        //translate section name
+        localizationService.localize("sections_" + rq.section).then(function (value) {
+            $scope.sectionName = value;
+        });
```

Comment:
I tried to localize the Members -> Member node ... so the ```#Member``` value.

In the line https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Web/Trees/MemberTreeController.cs#L132 I replaced the title ```memberType.Name``` with
```
(memberType.Name.StartsWith("#") ? new UmbracoHelper().GetDictionaryValue(memberType.Name.Substring(1)) : memberType.Name)
```
or 
```
(memberType.Name.StartsWith("#") ? Umbraco.GetDictionaryValue(memberType.Name.Substring(1)) : memberType.Name)
```


I also tried with the newer LocalizationService API:
```
(memberType.Name.StartsWith("#") ? Services.LocalizationService.GetDictionaryItemByKey(memberType.Name.Substring(1)).ItemKey : memberType.Name)
```

Then I had language for both Danish (da-DK) and English (en-US) .. and a dictionary item "Member" with value "Medlem" for Danish and "Member" for English. But no matter which language of those two I set for the user it seemed to get the dictionary item value in Danish. No sure if it knows the culture/language in this context?